scripts/coinweight.lua

```diff
@@ -48,7 +50,7 @@ local function createCoinsItem(nodeChar)
 	local nodeCoinsItem
 	if nodeChar.getParent().getName() == 'charsheet' then
 		nodeCoinsItem = DB.createChild(nodeChar.createChild('inventorylist'))
-		DB.setValue(nodeCoinsItem, 'name', 'string', 'Coins')
+		DB.setValue(nodeCoinsItem, 'name', 'string', COINS_INVENTORY_ITEM_NAME)
```

Comment:
Have the initialization of the item and the matcher use the same constant value.